packages/synapse-react-client/test/containers/entity/annotations/AdditionalPropertiesSchemaField.test.tsx

```diff
@@ -91,29 +104,15 @@ describe('AdditionalPropertiesSchemaField unit tests', () => {
     })
   })
 
-  describe('getWidgetFromPropertyType tests', () => {
-    it('gets TextWidget for integer', () => {
-      expect(getWidgetFromPropertyType(PropertyType.INTEGER)).toBe('TextWidget')
-    })
-
-    it('gets TextWidget for float', () => {
-      expect(getWidgetFromPropertyType(PropertyType.FLOAT)).toBe('TextWidget')
-    })
-
-    it('gets DateTimeWidget for datetime', () => {
-      expect(getWidgetFromPropertyType(PropertyType.DATETIME)).toBe(
-        'DateTimeWidget',
-      )
-    })
-
-    it('gets CheckboxWidget for boolean', () => {
-      expect(getWidgetFromPropertyType(PropertyType.BOOLEAN)).toBe(
-        'CheckboxWidget',
-      )
-    })
-
-    it('gets TextWidget for string', () => {
-      expect(getWidgetFromPropertyType(PropertyType.STRING)).toBe('TextWidget')
+  describe('getSchemaForPropertyType tests', () => {
```

Comment:
Instead of mapping custom annotation type to Widget, now we custom annotation type to schema, and let RJSF pick the widget based on the mapping we have already set up for regular schema properties.